npm/builder.js

```diff
@@ -141,10 +141,7 @@ Builder.prototype.release = function (releaseVersion) {
     function (callback) { builder.prepareRelease(releaseVersion, callback); },
     function (callback) { builder.build(callback); },
     function (callback) { builder.runTest(callback); },
-    function (callback) { builder.copyToDist(callback); },
     function (callback) { builder.commit(releaseVersion, callback); },
-    function (callback) { builder.publish(callback); },
-    function (callback) { builder.prepareNextIteration(callback); },
     function (callback) { builder.completeRelease(releaseVersion, callback); }
```

Comment:
If you want to easily manage PRs and contributors' remote, I recommend using my new tool [PRM](https://github.com/ldez/prm).

Perhaps you can use the environment variable `CI` to automate or not the `git push --tags` (https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables)
i.e. test if `CI` exist: if yes do `git push --tags`  otherwise print the command.